data/conf/rspamd/dynmaps/settings.php

```diff
@@ -97,15 +97,27 @@ function ucl_rcpts($object, $type) {
   global $pdo;
   $rcpt = array();
   if ($type == 'mailbox') {
+    // The mailbox itself
+    {

```

Comment:
Why putting it in { } ?

Reply:
I didnt want to pollute the scope but just noticed that PHP doesnt support that. I will remove them